locales/en/app.json

```
@@ -22,6 +22,7 @@
   "navigation": {
      "general": "to",
      "auth": "Your details",
+     "accessibilityStatement": "Accessibility Statement",
```

From content design, needs to be lowercase statement:```suggestion
     "accessibilityStatement": "Accessibility statement",
```

---

incorporated suggestion